src/components/ProfileView.jsx

```diff
@@ -28,7 +26,10 @@ import { AUTH_MODE } from 'actions/twoFactor'
 const PasswordSection = () => {
   const { t } = useI18n()
   const client = useClient()
-  const canAuthWithPassword = get(client, 'capabilities.can_auth_with_password')
+  const capabilities = client.capabilities
+  const canAuthWithPassword = capabilities
+    ? capabilities.can_auth_with_password
+    : true // If we have no capabilities, consider that we can login to the cozy with password
```

Comment:
you can also do 
```
const canAuthWithPassword = get(client, 'capabilities.can_auth_with_password', true /* If we have no capabilities, consider that we can login to the cozy with password*/)
```

Reply:
Thanks, I thought about that but was not sure if extra values would not be considered as extra keys. I've made the change 👍